### ISSUE TITLE
3.0.0-beta1 fails to build

### ISSUE THREAD

**Issue by mtsr** [kubernetes-client/java#327]
`[ERROR] Failed to execute goal on project client-java: Could not resolve dependencies for project io.kubernetes:client-java:bundle:3.0.0-beta1: The following artifacts could not be resolved: io.kubernetes:client-java-api:jar:3.0.0-SNAPSHOT, io.kubernetes:client-java-proto:jar:3.0.0-SNAPSHOT: Could not find artifact io.kubernetes:client-java-api:jar:3.0.0-SNAPSHOT -> [Help 1]
`

**Comment by Polpetta** [comment]
+1 Having the same issue here:
```
[davide@PcFissoDavide java]$ mvn clean compile -U
[INFO] Scanning for projects...
[INFO] Inspecting build with total of 5 modules...
[INFO] Installing Nexus Staging features:
[INFO]   ... total of 5 executions of maven-deploy-plugin replaced with nexus-staging-maven-plugin
[WARNING] The project io.kubernetes:client-java-parent:pom:3.0.0-beta1 uses prerequisites which is only intended for maven-plugin projects but not for non maven-plugin projects. For such purposes you should use the maven-enforcer-plugin. See https://maven.apache.org/enforcer/enforcer-rules/requireMavenVersion.html
[WARNING] The project io.kubernetes:client-java-api:bundle:3.0.0-beta1 uses prerequisites which is only intended for maven-plugin projects but not for non maven-plugin projects. For such purposes you should use the maven-enforcer-plugin. See https://maven.apache.org/enforcer/enforcer-rules/requireMavenVersion.html
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Build Order:
[INFO] 
[INFO] Kubernetes Client API
[INFO] client-java
[INFO] client-java-examples
[INFO] client-java-api
[INFO] client-java-proto
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] Building Kubernetes Client API 3.0.0-beta1
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ client-java-parent ---
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] Building client-java 3.0.0-beta1
[INFO] ------------------------------------------------------------------------
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-resources-plugin/maven-metadata.xml
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-resources-plugin/maven-metadata.xml (843 B at 2.3 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-install-plugin/maven-metadata.xml
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-install-plugin/maven-metadata.xml (623 B at 8.5 kB/s)
[WARNING] The POM for io.kubernetes:client-java-api:jar:3.0.0-SNAPSHOT is missing, no dependency information available
[WARNING] The POM for io.kubernetes:client-java-proto:jar:3.0.0-SNAPSHOT is missing, no dependency information available
Downloading from central: https://repo.maven.apache.org/maven2/junit/junit-dep/maven-metadata.xml
Downloaded from central: https://repo.maven.apache.org/maven2/junit/junit-dep/maven-metadata.xml (563 B at 8.2 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/nimbusds/lang-tag/maven-metadata.xml
Downloaded from central: https://repo.maven.apache.org/maven2/com/nimbusds/lang-tag/maven-metadata.xml (507 B at 7.0 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/nimbusds/nimbus-jose-jwt/maven-metadata.xml
Downloaded from central: https://repo.maven.apache.org/maven2/com/nimbusds/nimbus-jose-jwt/maven-metadata.xml (4.2 kB at 58 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/net/minidev/json-smart/maven-metadata.xml
Downloaded from central: https://repo.maven.apache.org/maven2/net/minidev/json-smart/maven-metadata.xml (849 B at 12 kB/s)
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] Kubernetes Client API .............................. SUCCESS [  0.043 s]
[INFO] client-java ........................................ FAILURE [  1.084 s]
[INFO] client-java-examples ............................... SKIPPED
[INFO] client-java-api .................................... SKIPPED
[INFO] client-java-proto .................................. SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 1.793 s
[INFO] Finished at: 2018-07-31T12:15:35+02:00
[INFO] Final Memory: 20M/287M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project client-java: Could not resolve dependencies for project io.kubernetes:client-java:bundle:3.0.0-beta1: The following artifacts could not be resolved: io.kubernetes:client-java-api:jar:3.0.0-SNAPSHOT, io.kubernetes:client-java-proto:jar:3.0.0-SNAPSHOT: Could not find artifact io.kubernetes:client-java-api:jar:3.0.0-SNAPSHOT -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :client-java
```

**Comment by mtsr** [comment]
If you need to move forward right now, as I did, you can clone the repo and remove -beta1 from the version names. Maven can’t handle -beta1-SNAPSHOT.  

After that you can `mvn install` locally and use that. 

--

Jonas

**Comment by Polpetta** [comment]
Wow! Thanks! :+1:

**Comment by brendandburns** [comment]
oops, my bad, I will get this fixed...

**Comment by brendandburns** [comment]
Fixed by #330

**Comment by nsidhaye** [comment]
I tried to use this in one of gradle project by  using `compile 'io.kubernetes:client-java:2.0.0-beta1'` and it was mere failure 

```
Could not find io.kubernetes:client-java-api:3.0.0-SNAPSHOT.
```

This will fix above issue. Obviously for time being we can work on master branch.
